config/routes.rb

```diff
@@ -10,9 +10,10 @@
     get "/users/sign_out", :to => "sessions#destroy"
   end
 
+ 
   # This must be the very last route in the file because it has a catch-all route for 404 errors.
   # This behavior seems to show up only in production mode.
   root to: 'homepage#index'
-
+  mount HydraEditor::Engine => '/'
```

Comment:
Sure, I was reading the readme file at https://github.com/jechols/hydra-editor/blob/master/README.md and it suggested to mount the routes in config/routes.rb. I think we might not need to mount it if we decide not to use the default paths.